### PR TITLE
JS Capstone

### PR DESCRIPTION
**Pull request by tkz96**
This PR is the entire project made in JavaScript for a TV database.

Link to the video presentation:

https://drive.google.com/file/d/1UMJe8Ns_7b5FHUQo36gwxdlzRTGoCjW9/view?usp=sharing

### CODE REVIEW

File: src/apiclass.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/no-cycle */
+
+import { displayEpisodes, displayComments } from './displayItems.js';
+
+const ourID = 'VquE0tA2WmPqHydxagQA';
+
+const episodes = ' https://api.tvmaze.com/seasons/1/episodes';
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
```
Comment (mateo951) [inline src/apiclass.js:8]: - The web app uses an API that bases its development around it ✔
- The API should allow for getting a list of items with a unique item id and detailed information about the items ✔
- The project uses the Involvement API to record the different user interactions (likes and comments) 📝
 
Well done 🙌🔥

File: src/displayItems.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      <div class="info-card">
+        <figure>
+          <img src="${element.image.medium}" alt="">
+          <figcaption>${element.name}</figcaption>
+        </figure>
+        <div class="${element.id}">
+        <img id='likesImage' class="heart" src="https://yusufnviiri.github.io/capTest/images/hearts.png" alt="">
+              </div>
+        <span class="likes" id="${element.id}">0 likes</span>
+        <button class="comments" data-id="${element.id}">Comments</button>
```
Comment (mateo951) [inline src/displayItems.js:80]: Popup is working beautifully ✔ Congrats 👌

image

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @omar25ahmed  and @ryxtor, 

📝👌 **Your project highlights** 😎 
- General Requirements 💪🤓 (Gitflow 🔥)
- Implementation of API's 🤯

### Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## REQUIRED CHANGES

- [ ] As this is a GROUP CAPSTONE - while working, you were supposed to give reviews to each other and merge their feature branches into the dev branch. Be sure to apply this change in your next GROUP CAPSTONE 🤓 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by mateo951** [comment]
- [ ] Kindly add a video presenting the features of the project 🔥
- [ ] Please be sure to add the link to your **VIDEO PRESENTATION** in your PR description summary 🧐

**Comment by tkz96** [comment]
Required changes have been noted.
Video link added to the description of the summary.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @tkz96 and @yusufnviiri 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**